src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -328,6 +328,11 @@
     };
     const scrollToInvalidInput = () => {
         const firstInvalidInput = editForm.querySelector('.ibexa-input.is-invalid');
+
+        if (!firstInvalidInput) {
```

Comment:
Why scrollToInvalidInput is invoked if there is no invalid input?

Reply:
As for HTML, there are no nodes with the selector `.ibexa-input.is-invalid` because the error concerns the lack of added Field definitions
![Screenshot 2022-08-08 at 09 27 11](https://user-images.githubusercontent.com/24355391/183363565-18ca543c-1f2b-47f3-908e-cc50235f7916.png)